src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The Duke class contains the main method to update the TaskList and
+ * execute the user's commands and input.
+ */
+class Duke {
+    public final static String LINE = "__________________________________________";
```

Comment:
Personally, I feel that all UI in/out variables, content should be stored in your ui class